pkg/controllers/provisioning/suite_test.go

```diff
@@ -52,8 +52,8 @@ var _ = BeforeSuite(func() {
 	env = test.NewEnvironment(ctx, func(e *test.Environment) {
 		cloudProvider := &fake.CloudProvider{}
 		registry.RegisterOrDie(ctx, cloudProvider)
-		controller = provisioning.NewController(ctx, e.Client, corev1.NewForConfigOrDie(e.Config), cloudProvider)
-		scheduler = scheduling.NewController(e.Client, controller)
+		provisioners = provisioning.NewController(ctx, e.Client, corev1.NewForConfigOrDie(e.Config), cloudProvider)
```

Comment:
Why the plural variable name if you're only creating one provisioner? I noticed this in expectations.go, too, and was curious.

Reply:
provisioners is the name I've been using for the controller that manages provisioners. I could use something like provisionerController, but 🤷